randovania/game/game_enum.py

```diff
@@ -1,3 +1,6 @@
+# mypy: disable-error-code="no-redef"
+# we specifically abuse redefinition in this class to import game_module
```

Comment:
Huh, I assumed there was some way to do what we have here without redefining game_module.

Reply:
there is, but we do it this way to avoid importing until runtime. it's necessary